observations/forward_operators/DEFAULT_obs_def_mod.F90

```diff
@@ -839,7 +839,7 @@ subroutine write_obs_def(ifile, obs_def, key, fform)
          source, revision, revdate, text2='observation type '//trim(get_name_for_type_of_obs(obs_def%kind)))
    endif
    if (is_ascii) then
-      write(ifile, 12) obs_def%ens_size, obs_def%external_FO_key
+      write(ifile, 12) obs_def%ens_size, key
```

Comment:
remind me why this change is needed?  is this the main obs key (1,2,3, etc in the sequence regardless of obs type) or is this the per-type key?  i know it will get ignored on read, but for tracking problems is this going to be per-type key that would be printed out by debugging statements during a run?

Reply:
this is now a merged pull request, but did anyone answer why the key on this line changed?